### PR TITLE
Update description for launching offer

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

**Review by zkayyali812 — APPROVED** [review]

**Review by ianf77 — COMMENTED** [review]
One change

File: stories/topics/con-gcp-application-deploy.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 = Application deployment
 
-Now that you have launched the public or private offer, you can begin configuration before deploying the application.
+To lauch your offer, on the Google Cloud Console, navigate to the marketplace and search for *Red Hat Ansible Automation Platform 2 - Up to 100 Managed Nodes*. After selecting this offer click btn:[Launch].
```
[the remark this answers is not in the document]
Comment (prat98) [reply stories/topics/con-gcp-application-deploy.adoc]: fixed!

[the remark this answers is not in the document]
Comment (ianf77) [reply stories/topics/con-gcp-application-deploy.adoc]: To launch your offer on the Google Cloud Console,  (remove the extra comma, check spelling)

**Review by ianf77 — COMMENTED** [review]
One comment.

File: stories/topics/con-gcp-application-deploy.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 = Application deployment
 
-Now that you have launched the public or private offer, you can begin configuration before deploying the application.
+To lauch your offer on the Google Cloud Console, navigate to the marketplace and search for *Red Hat Ansible Automation Platform 2 - Up to 100 Managed Nodes*. After selecting this offer click btn:[Launch].
```
[the remark this answers is not in the document]
Comment (prat98) [reply stories/topics/con-gcp-application-deploy.adoc]: good catch! Fixed typo thanks!

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**